Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -251,6 +252,11 @@
   - type: Tag
     tags:
     - Write
+  - type: Reflect
+    reflectProb: 0.3 #Goob rebalace
```

```suggestion
    reflectProb: 0.25 #Goob rebalace
    otherTypeReflectProb: 0
```
I changed EnergyDaggerLoud proto but forgot about this one.